File: 3172-divisible-and-non-divisible-sums-difference/divisible-and-non-divisible-sums-difference.java
class Solution {
    public int differenceOfSums(int n, int m) {
        int num1=0;
        int num2=0;
        int i=1;
        while(i<=n){
            if(i % m == 0)
            num2 += i;
            else{
                num1+=i;
            }
           i++;
        }
        return num1 - num2;
    }
}

File: 3172-divisible-and-non-divisible-sums-difference/divisible-and-non-divisible-sums-difference.c
int differenceOfSums(int n, int m) {
    int i=1;
    int num1=0,num2=0;
    while(i<=n){
        if( i % m == 0){
            num2+=i;
        }
        else{
            num1+=i;
        }
        i++;
    }
    return num1-num2;
}